tamr_toolbox/dataset/manage.py

```diff
@@ -0,0 +1,381 @@
+from typing import List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.dataset.resource import Dataset
+from tamr_unify_client.attribute.resource import AttributeSpec
+from tamr_unify_client.attribute.type import AttributeType
+
+from tamr_toolbox.models.data_type import JsonDict
+
+
+def exists(*, client: Client, dataset: str) -> bool:
```

Comment:
I think you should change this to dataset_name for consistency. 
```suggestion
def exists(*, client: Client, dataset_name: str) -> bool:
```